src/data/users/balanced02.json

```diff
@@ -0,0 +1 @@
+{"username":"Balanced02","name":"Adepoju Daniel","location":"Abuja, Nigeria","avatar":"https://pbs.twimg.com/profile_images/1272287309615030272/qXE_sg-__400x400.jpg","currentRole":"Software Engineer","currentCompany":"Interswitch","portfolio":"https://github.com/Balanced02","bio":"Madly in love with JavaScript","category":"developer","social":{"twitter":"Balanced02","linkedIn":"balanced02","github":"Balanced02","email":"adebalanced02@gmail.com"},"skills":["React","React Native","Javascript","NodeJs","MongoDB"],"selectedWorks":[{"projectTitle":"Edubase","projectImageUrl":"https://edubaseforschools.com/wp-content/uploads/2020/05/stud_class.png","projectDescription":"A cloud based Virtual Learning Environment. Customized and branded for your school with special features which includes Classroom Management, Content, Resources, Result Management and lots more","projectLink":"https://www.edubase.app"},{"projectTitle":"VCF Tel Extractor","projectImageUrl":"https://upload.wikimedia.org/wikipedia/commons/thumb/d/db/Npm-logo.svg/1200px-Npm-logo.svg.png","projectDescription":"Openn source module to extract phone numbers from .vcf file","projectLink":"https://www.npmjs.com/package/vcftelextractor"}],"id":6}
```

Comment:
you're not supposed to add the ID yourself. The generator adds it when you make changes

Reply:
The generator doesn't add it, This is supposed to be in my next PR, cause I noticed that and fixed it after I made this PR

Reply:
Kindly check the updated codes, I've made a push

Reply:
Hmmm

Reply:
Let me investigate this

Reply:
@Kunoacc Welldone, Sherlock Holmes...